src/headers/syscheck_op.h

```diff
@@ -25,6 +25,7 @@
 #define SK_GNAME   8
 #define SK_INODE   9
 #define SK_NFIELDS 10
+#define SK_SHA256  11
```

Comment:
This definition exceeds the field size limit.

`SK_NFIELDS` is the number of fields, now there will be 11 fields. This parameter is used to evaluate all fields: https://github.com/wazuh/wazuh/blob/master/src/shared/syscheck_op.c#L104-L106

Please swap `SK_NFIELDS` and  `SK_SHA256`:
```c
#define SK_SHA256 10
#define SK_NFIELDS 11
```